inputs/AccountSettings.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { Text, View, TextInput, TouchableOpacity, ScrollView, SafeAreaView } from 'react-native';
+import styles from '../styles';
+
+export default function UserSettings() {
+    const [newPassword, setNewPassword] = useState('');
+    const [confirmPassword, setConfirmPassword] = useState('');
+    const [currentPassword, setCurrentPassword] = useState('');
+    const [newEmail, setNewEmail] = useState('');
+
+    const handleChangePassword = () => {
+        // write the logic to change password
+    };
+
+    const handleChangeEmail = () => {
+        // write the logic to change email
+    };
+
+    const handleDeleteAccount = () => {
+        // write the logic to delete account
+    };
+
+    const handleLogout = () => {
+        // write the logic to logout
+    };
+
+    return (
+        <ScrollView contentContainerStyle={styles.container}>
+            <Text style={styles.title}>Account Settings</Text>
+            <View style={styles.sectionContainer}>
+                <Text style={styles.title}>Change Password</Text>
+                <View style={styles.inputContainer}>
+                <TextInput
+                    style={styles.input}
+                    onChangeText={setNewPassword}
+                    value={newPassword}
+                    placeholder="New Password"
+                    secureTextEntry={true}
```

Comment:
When the prop is a boolean and it is not changing, you do not necessarily have to explicitly say ``{true}`` and just adding ``secureTextEntry`` without a value works, but is fine for now for readability